wiki/osu!_rank_one_history/en.md

```diff
@@ -0,0 +1,265 @@
+# History of osu! Rank 1 Players
+
+The following is a list of players who are recognized to have reached rank 1 in the [osu! Global Rankings](https://osu.ppy.sh/rankings/osu/performance).
+
+Over the years the osu! global rankings has used three different systems for ranking players:
+
+- [Ranked Score](/wiki/score#ranked-score) (July 2007 - April 2012)
+- [ppv1](/wiki/Performance_points/ppv1) (April 2012 - 27 January 2014)
+- [ppv2](/wiki/Performance_points) (Since 27 January 2014)
+
+Note: During the period where Ranked Score was used, Rank 1 would often change as of such this list only includes users who have been verified to have reached Rank 1. Early dates use months where exact dates aren't available.
+
+## 2007
+
+| Player | Date |
+| --: | :-- |
+| ![][flag_AU] [peppy](https://osu.ppy.sh/users/2) | 2007-10-4 |
+| ![][flag_AU] [eyup](https://osu.ppy.sh/users/88) | October 2007 |
+| ![][flag_US] [Ivalset](https://osu.ppy.sh/users/827) | 2007-10-11 |
+| ![][flag_US] [Saturos](https://osu.ppy.sh/users/3781) | 2007-12-24 |
+
+## 2008
+
+| Player | Date |
+| --: | :-- |
+| ![][flag_US] [Cyclone](https://osu.ppy.sh/users/18589) | 2008-10-07 |
+| ![][flag_US] [Rolled](https://osu.ppy.sh/users/5243) | 2008-12-11 |
+
+## 2009
+
+| Player | Date |
+| --: | :-- |
+| ![][flag_US] [Cyclone](https://osu.ppy.sh/users/18589) | 2009-01-19 |
+| ![][flag_JP] [tobebuta](https://osu.ppy.sh/users/14053) | January 2009 |
+| ![][flag_GB] [Doomsday](https://osu.ppy.sh/users/18983) | March 2009 |
+| ![][flag_US] [Cyclone](https://osu.ppy.sh/users/18589) | May 2009 |
```

Comment:
```suggestion
| ![][flag_JP] [tobebuta](https://osu.ppy.sh/users/14053) | March 2009 |
| ![][flag_US] [Cyclone](https://osu.ppy.sh/users/18589) | March 2009 |
| ![][flag_PL] [Kubu](https://osu.ppy.sh/users/29130) | April 2009 |
| ![][flag_US] [Cyclone](https://osu.ppy.sh/users/18589) | April 2009 |
```
- tobebuta was not rank 1 in January 2009: on 21 Januray 2009 in the archived leaderboard Cyclone is rank 1 with 3,484,062,074 ranked score. Meanwhile by 4 February he is still rank 1 and at that point tobebuta has a ranked score of 3,333,333,333, meaning it's impossible that tobebuta passed Cyclone in score in Januray after 21 Januray, making him being rank 1 in January impossible.

- Cyclone was rank 1 throughout the entirety of February 2009, as can be seen by checking every archived leaderboard from that period.

- Sometime between 9 March and 18 March tobebuta briefly overtook Cyclone as rank 1, before Cyclone took over again (as is proved by the downwards arrow next to their rank on the 18 March leaderboard archive).

- kuburaczek (now known as Kubu) was rank 1 in April 2009, getting overtaken by Cyclone afterwards as can be attributed by the leaderboard of 18 April and by this archive of his profile: https://web.archive.org/web/20090416172022/http://osu.ppy.sh/u/29130

- While I can't be definitive with this, it's highly unlikely Doomsday was rank 1 during this period, considering his position in the leaderboards in March and in April, neither of which support in the slightest he reached rank 1 at this time.

Archive.org leaderboards (If certain dates are refered, this is likely the source): https://web.archive.org/web/20090501000000*/http://osu.ppy.sh/p/playerranking